examples/layers/preprocessing/bounding_box/demo_utils.py

```diff
@@ -64,11 +64,17 @@ def visualize_data(data, bounding_box_format):
 def visualize_bounding_boxes(image, bounding_boxes, bounding_box_format):
     color = np.array([[255.0, 0.0, 0.0]])
     bounding_boxes = bounding_box.to_dense(bounding_boxes)
+    bounding_boxes = bounding_box.convert_format(
```

Comment:
Why convert to yxyx at all? Can't we just convert to rel_yxyx directly?

Reply:
what you just commented is turning out to be one of the most difficult issues to catch in KerasCV code bases.

calling to_dense() actually changes image sizes, so if you are in `rel_xywh` this causes nonsense boxes.

Wonder if we can catch this somehow

Reply:
Ack -- please open an Issue about this